_includes/footer.html

```diff
@@ -0,0 +1,51 @@
+<!-- create a footer -->
+<footer class="footer">
+  <div class="row mx-0">
+    <div class="col-md-5 left-div">
+      <a href="{{ '/' | relative_url }}">
+        <div class="footer-logo">
+          <img src="./assets/img/mlh-prep.png" />
+        </div>
+      </a>
+
+      <div class="social-links">
+        <a href="https://www.linkedin.com/school/mlh-fellowship/" class="links"
+          ><i class="fab fa-linkedin fa-lg"></i
+        ></a>
+        <a href="https://www.instagram.com/mlhacks/" class="links"
+          ><i class="fab fa-instagram fa-lg"></i
+        ></a>
+        <a
+          href="https://twitter.com/MLHacks?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"
+          class="links"
+          ><i class="fab fa-twitter fa-lg"></i
+        ></a>
+        <a href="https://www.facebook.com/MajorLeagueHacking" class="links"
+          ><i class="fab fa-facebook fa-lg"></i
+        ></a>
+        <a href="https://www.youtube.com/c/Majorleaguehacking" class="links"
+          ><i class="fab fa-youtube fa-lg"></i
+        ></a>
+      </div>
+    </div>
+
+    <div class="col-md-6 col-12 right-div">
+      <div class="list">
+        <p>
+          <a href="/#">Blog</a>
+        </p>
+        <p>
+          <a href="/#about">About</a>
```

Comment:
Where is the #about leading to?

Reply:
For now, it's leading nowhere, We were thinking of linking it to the '**description section**' once the sections were developed. Should I remove it?

Reply:
I think we can let it be there then👍